updates/builder_table_update_ipricegroup_ocapiplugin_resources_add_route_action.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace IPriceGroup\OcApiPlugin\Updates;
+
+use Schema;
+use October\Rain\Database\Updates\Migration;
+
+class BuilderTableUpdateIpricegroupOcapipluginResourcesAddRouteAction extends Migration
```

Comment:
Do you mean `RouterMethod` instead of `RouteAction`?

Also, the filename should say `router_method` instead of `route_action`.